LOCAL_VIRTUALENV.rst

```diff
@@ -91,22 +91,41 @@ To use your IDE for Airflow development and testing, you need to configure a vir
 environment. Ideally you should set up virtualenv for all Python versions that Airflow
 supports (3.5, 3.6).
 
-Consider using one of the following utilities to create virtual environments and easily
-switch between them with the ``workon`` command:
-
-- `pyenv <https://github.com/pyenv/pyenv>`_
-- `pyenv-virtualenv <https://github.com/pyenv/pyenv-virtualenv>`_
-- `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_
-
 To create and initialize the local virtualenv:
 
 1. Create an environment as follows:
 
-   ``mkvirtualenv <ENV_NAME> --python=python<VERSION>``
+   a. Consider using one of the following utilities to create virtual environments and easily switch between them with the ``workon`` command:
+
+    - `pyenv <https://github.com/pyenv/pyenv>`_
+    - `pyenv-virtualenv <https://github.com/pyenv/pyenv-virtualenv>`_
+    - `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_
+
+    ``mkvirtualenv <ENV_NAME> --python=python<VERSION>``
+  
+   b. Create a local virtualenv with conda
```

Comment:
This structure suggest that I should follow point a) then b) and that's not true.

Reply:
Would it be better if I change the ordered list into unordered list? My intention is to group the two ways of creating virtualenvs: using virtualenvwrapper etc vs conda into one big section as creating virtualenvs, as one equvilant to the other. So we don't need to repeat pip install and DB init separately.

Reply:
Reorganize with a cleaner structure. Let me know how you think.

Reply:
@nuclearpinguin A kindly reminder :). PTAL. Thanks!